main/mySpace/access_details_session.php

```diff
@@ -267,12 +267,13 @@ function getReport($userId, $from, $to, $addTime = false)
 
     $courseSessionTable = '';
     $courseSessionTableData = [];
+    $iconCourse = Display::return_icon('course.png',null,[],ICON_SIZE_SMALL);
     foreach ($report as $sessionId => $data) {
         foreach ($data['courses'] as $courseId => $courseData) {
             if (empty($courseData)) {
                 continue;
             }
-            $courseSessionTable .= Display::page_subheader3($data['name'][$courseId]);
+            $courseSessionTable .= '<div class="data-title">'.Display::page_subheader3($iconCourse.$data['name'][$courseId]).'</div>';
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Line exceeds maximum limit of 120 characters; contains 134 characters](https://app.codacy.com/manual/chamilo/chamilo-lms/pullRequest?prid=5266914)